git_test.go

```diff
@@ -159,6 +164,101 @@ func TestCloneRepo(t *testing.T) {
 	}
 }
 
+func TestCloneRepoSSH(t *testing.T) {
+
```

Comment:
```suggestion
```
Maybe we should introduce [gofumpt](https://github.com/mvdan/gofumpt) in this repo. 🤔